src/defu.ts

```diff
@@ -7,10 +7,6 @@ type defuObj = {
 }
 
 function _defu<T extends defuObj> (baseObj: T | any, defaults: T | any): T {
-  if (!isObject(baseObj)) {
```

Comment:
Use case of this line was to handle conditions when the user is not providing options

Reply:
Actually even with the tests it never goes inside of it, because of using `reduce` with a default value.